### ISSUE TITLE
Problem after image upload

### ISSUE THREAD

**Issue by luckygala** [Laverna/laverna#955]
when an image was uploaded into a note and note was saved, i can't view content of note.

console log error:
TypeError: Argument 1 is not valid for any of the 1-argument overloads of URL.createObjectURL.
markdown-it-file.js:8:813

**Comment by ghost** [comment]
Yeah, same issue. Cannot view the contents after uploading a image. The note's title is visible but the content has vanished.
